editor.planx.uk/src/pages/FlowEditor/floweditor.scss

```diff
@@ -47,7 +47,8 @@ $fontMonospace: "Source Code Pro", monospace;
   li {
     list-style: none;
     padding: 0;
-    margin: 0;
+    margin: 0 auto;
+    width: max-content;
```

Comment:
This is fixing the overlap but reducing 100% width of section components (needed for dashed divider line). There is an appended class for section list items (`type-Section`) so adding 100% width for that class should fix it.